tests/ui-nightly/transmute-ref-alignment-increase.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
+  --> tests/ui-nightly/transmute-ref-alignment-increase.rs:15:35
+   |
+15 | const INCREASE_ALIGNMENT: &AU16 = transmute_ref!(&[0u8; 2]);
+   |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: source type: `MaxAlignsOf<[u8; 2], AU16>` (16 bits)
+   = note: target type: `AlignOf<[u8; 2]>` (8 bits)
```

Comment:
In the implementation of `transmute_ref`, is it possible to reverse the order of the alignment-checking transmute, such that `AlignOf<[u8; 2]` is the 'source type' in this error message?

Reply:
Good idea! Done.